### ISSUE TITLE
Create the Puppet master base box

### ISSUE THREAD

**Issue by gnufred** [gnufred/crypet#2]
The Puppet master can only run on Linux so we need to create a different box that Vagrant can manage.

We can use any Linux distro but here's the one I am considering so far:

### Ubuntu 16.04LTS (and 18.04 when available).
Stable, reliable.

### Archlinux
We would have the latest Puppet version, unlike Ubuntu. But it's not a stable system. Unless with specify which package version we want, we will always get the latest versions.

**Comment by binbrun** [comment]
Ubuntu. Stability is above everything. Nothing worse than having an unstable system.

**Comment by cdchris12** [comment]
Agreed. Ubuntu is reliable and we'll supported by the community at large if we run into problems

**Comment by gnufred** [comment]
Ubuntu it is then. I'll try to grab 18.04 release candidates or beta if possible.
